src/dwarf.cc

```diff
@@ -60,6 +61,15 @@ int DivRoundUp(int n, int d) {
   return (n + (d - 1)) / d;
 }
 
+namespace {
+
+// uint64/32 max is a tombstone value added by https://reviews.llvm.org/D81784.
+bool IsValidDwarfAddress(uint64_t addr, bool dwarf64) {
+  return addr != 0 && addr != (dwarf64 ? std::numeric_limits<uint64_t>::max()
```

Comment:
DWARF64 actually refers to a different encoding format. It is orthogonal to the address space size of the host.

You'll want to check `address_size == 4`

Reply:
Done, PTAL